### ISSUE TITLE
Catalyst currently doesn't get any beautiful context menus

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#9362]
### Description

When using the Flyout Context Menus samples on Catalyst if you initiate the "Right-click to see beautiful menus" Context Menu all you get are boring non-beautiful looking menus

**Comment by Eilon** [comment]
Yeah it's true, but also currently a limitation of MacCatalyst. Icons are not supported. Funnily, iOS does support them!

**Comment by kubaflo** [comment]
Closing as external/not planned for the MAUI backlog. This request is blocked by platform limitations and has remained unselected for multiple releases. If the upstream platform capabilities change and there is a current MAUI implementation path, please file a fresh issue.

**Comment by kubaflo** [comment]
Reopening — this issue should not have been closed.

It carries `partner/cat 😻` (a partner team is tracking it). Those labels mean the report was already confirmed as a real product defect, so it is not eligible for backlog cleanup closure. Closure here requires specific fix evidence (a merged PR that closes it, or a `fixed-in-*` label), and there was none.

Sorry for the noise, and thanks to everyone who flagged it.

> Disclosure: this comment was generated by AI.
